pkgs/applications/version-management/gitlab/gitaly/deps.nix

```diff
@@ -1319,8 +1319,8 @@
     fetch = {
       type = "git";
       url = "https://github.com/ugorji/go";
-      rev = "d75b2dcb6bc8";
-      sha256 = "0di1k35gpq9bp958ywranpbskx2vdwlb38s22vl9rybm3wa5g3ps";
+      rev = "v1.1.4";
```

Comment:
I'd like to get a quick confirmation from @flokli that this undoing this change from https://github.com/NixOS/nixpkgs/pull/80062 is fine.

Reply:
Hm, yeah, it seems a little strange..

Reply:
Seems like it was not done on purpose so it should be fine

Reply:
As written there, I don't know why `vgo2nix` it moved to that rev (maybe another package wasn't compatible with it?!), but that rev seems to be older than 1.1.4.

If it now resolves to 1.1.4 again, works for me :-)